app/assets/stylesheets/application.css

```
@@ -39,7 +57,7 @@ body {
 a {
   background-color: #6dca7f;
   color: var(--black);
-  color: rgb(61, 146, 201);
+  color: 3D92C9;
```

@JacOng17 oops, missing a # here e.g. ``
```suggestion
  color: #3D92C9;
```